graphql2/graph/_Query.graphqls

```diff
@@ -0,0 +1,132 @@
+
+type Query {
```

Comment:
Same as mutation, cut and paste with no changes other than removing the gql API key stuff